Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -869,6 +872,9 @@ static bool validateCopyTextureToTexture(const WGPUImageCopyTexture& source, con
         // https://developer.apple.com/documentation/metal/mtlblitcommandencoder/1400756-copyfromtexture?language=objc
         // "When you copy to a 1D texture, height and depth must be 1."
         auto sourceSize = MTLSizeMake(copySize.width, 1, 1);
+        if (anyDimensionIsZero(sourceSize))
```

Comment:
I'm confused. You just said the height and depth are 1. So why are we checking all the dimensions? Shouldn't we just check if copySize.width is 0?

Reply:
What happens when we don't take this early-out? Does Metal have a similar early-out inside the framework? Or does it assert?

Reply:
Metal asserts at least when running with the debug device. Maybe ultimately a non-issue however it breaks running with the debug device which is useful for development.

Good idea on the checks, I will just list them explicitly.